code/obj/machinery/nuclearbomb.dm

```diff
@@ -7,6 +7,7 @@
 	anchored = 0
 	event_handler_flags = IMMUNE_MANTA_PUSH
 	var/health = 150
+	var/max_health = 150
```

Comment:
isn't there already a var for health and max health

Reply:
oh yeah, guess that happened after the bomb was written.